phoenicis-configuration/src/main/java/org/phoenicis/configuration/localisation/PropertiesResourceBundle.java

```diff
@@ -0,0 +1,24 @@
+package org.phoenicis.configuration.localisation;
+
+import java.util.Enumeration;
+import java.util.Properties;
+import java.util.ResourceBundle;
+
+public class PropertiesResourceBundle extends ResourceBundle {
```

Comment:
I don't think so. Check PlayOnLinux/Scripts#258 for how it works.
To translate the json files, I am planning a script which extracts the relevant Strings.